common/src/main/java/gov/cms/ab2d/common/model/Attestation.java

```diff
@@ -20,7 +23,8 @@
 public class Attestation {
 
     @Id
-    @GeneratedValue
+    @GeneratedValue(strategy = SEQUENCE, generator = "attestation_id_seq")
+    @SequenceGenerator(name = "attestation_id_seq", sequenceName = "attestation_id_seq", allocationSize = 1)
```

Comment:
Same here. Can we let JPA pick the default values?

Reply:
I tried to use simply the `@GeneratedValue` hoping JPA would pick the defaults.

But after I got my liquibase changesets working properly, when I tried to run the spring boot app, it started giving the following error in a stack trace : 

```
missing sequence [hibernate_sequence]
```

Since the primary key column is auto-incrementing,  hibernate needs to know how to increment it from 1, 2, 3 .... 

My first attempt at overcoming this was to use `strategy = 'IDENTITY'` which seemed simpler as it was a single property to the (existing) `@GeneratedValue` annotation.

But after doing some more research on it, I found a few articles recommending `SEQUENCE`  over `IDENTITY` (especially for PostgreSQL). Here are a couple of articles
[1](https://thoughts-on-java.org/hibernate-postgresql-5-things-need-know/) 
[2](https://vladmihalcea.com/hibernate-identity-sequence-and-table-sequence-generator/)

When liquibase creates the primary key for the tables, it automatically creates the corresponding _sequence generator_ for each of the tables.  So to get the `SEQUENCE` strategy working, I needed to let hibernate know the respecgive name of the `sequence_generator` for each of the `entities`. I keyed in the name of the PostgreSQL generated sequence numbers and it started working fine.   

If there is a better way to do this, I am happy to change it.

Reply:
@spathiyil Just stick `CREATE SEQUENCE hibernate_sequence` into your first Liquibase migration and you'll be good to go with the defaults.

Reply:
done. created `hibernate_sequence` via liquibase, so that hibernate can use this as the default generator for generating the primary key for all the tables.